src/requests/user.js

```diff
@@ -6,6 +6,7 @@
  */
 
 // Dependencies
+require('dotenv').config()
```

Comment:
do we need this here ?

Reply:
yes , else it won't take the userservicehost and url @nevil-mathew